docs/source/documentation.rst

```diff
@@ -168,12 +168,18 @@ GPU-compatible installation
 ---------------------------
 .. NOTE :: This feature is not available if you are using a macOS.
 
+Linux and Windows 10
```

Comment:
Is there a specific reason for only specifying "Linux and Windows 10"? Is it because that we have tested these instructions on a GPU machine having base OS "Linux and Windows 10" only?

Reply:
I specified Linux and Windows 10 because I don't think there are many other options.
CUDA requires an Nvidia GPU and there haven't been any MacOS products with Nvidia GPUs in the last few years. Users could use an external GPU but I've never played with this kind of hardware.

Reply:
I guess we shouldn't be very specific about only "Linux and Windows 10". Anyways, not a big issue, so I'm approving this PR. Everything looks good to me.